plugins/TagFix_MultipleTag_Lang_fr.py

```diff
@@ -64,7 +64,7 @@ def node(self, data, tags):
         if ((not "highway" in tags and not "public_transport" in tags and "leisure" not in tags and ("type" not in tags or tags["type"] != "associatedStreet")) and
             (self.SalleDesFetes.match(tags["name"]) or self.MaisonDeQuartier.match(tags["name"])) and
             not ("amenity" in tags and tags["amenity"] == "community_centre")):
-            err.append({"class": 3032, "subclass": 3, "text": T_(u"Put a tag for a village hall or a community center"),
+            err.append({"class": 3032, "subclass": 3, "text": T_(u"Put a tag for a village hall or a community centre"),
```

Comment:
Suggestion: 
Add a tag for a [...]

Reply:
I would change this to "**The name indicates this is** a village hall or community centre"

Reply:
@jocelynj @frodrigo just ccing as this needs your input IMO

Reply:
> Google suggests Salle des fetes means 'party hall'

Yes, that the issue of mapping this kind of french amenity. The name really mean 'party hall'. But it is not the usage uniq usage. It is a multi usage hall on village The name depend on municipality.

Reply:
So @danieldegroot2 I guess your suggestion is also fine. (Maybe 'suggests' rather than 'indicates' to make it slightly less strong, but I'll leave that to you)